Docker.DotNet/Endpoints/MiscellaneousOperations.cs

```diff
@@ -64,32 +63,50 @@ public Task<Stream> MonitorEventsAsync(ContainerEventsParameters parameters, Can
             }
 
             JsonRequestContent<Config> data = parameters.Config == null ? null : new JsonRequestContent<Config>(parameters.Config, this.Client.JsonSerializer);
-                        
+
             IQueryString queryParameters = new QueryString<CommitContainerChangesParameters>(parameters);
             DockerApiResponse response = await this.Client.MakeRequestAsync(this.Client.NoErrorHandlers, HttpMethod.Post, "commit", queryParameters, data).ConfigureAwait(false);
             return this.Client.JsonSerializer.DeserializeObject<CommitContainerChangesResponse>(response.Body);
         }
 
         public Task<Stream> GetImageAsTarballAsync(string name, CancellationToken cancellationToken)
         {
-            if (string.IsNullOrEmpty(name))
+            return GetMultipleImagesAsTarballAsync(new[] { name }, cancellationToken);
```

Comment:
Does this change the output tar? IE: Is the call to multiple "a tar of image tar's" or in the case its just 1 is it a single image tar?